index.js

```diff
@@ -253,6 +254,15 @@ module.exports = (command, args, options) => {
 		}
 	});
 
+	processDone.cancel = function () {
+		if (canceled) {
+			return;
+		}
+
+		canceled = true;
+		spawned.kill();
```

Comment:
Probably this should actually be:

```js
if (spawned.kill()) {
  canceled = true;
}
```

Otherwise trying to `cancel()` an already killed process will set it `canceled` although `cancel()` did nothing.

See Node source code: https://github.com/nodejs/node/blob/master/lib/internal/child_process.js#L435